### PR TITLE
Add Turbine client endpoint

### PR DESCRIPTION
**Pull request by adamkorynta**
Implemented Turbine related test cases and classes. Also made updates to embankments to include the LocationIdentifier and updated cwms-radar-model yaml.

fixed formatting in the template file to minimize the need to reformat after generating

### CODE REVIEW

**Comment by adamkorynta** [comment]
Based on PR: https://github.com/USACE/cwms-data-api/pull/717

File: cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/LocationIdentifier.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+/**
+ * LocationIdentifier
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@jakarta.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2024-06-19T14:24:42.475228900-07:00[America/Los_Angeles]")
+public class LocationIdentifier {
```
Comment (rma-bryson) [inline cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/LocationIdentifier.java:37]: Why do we need a LocationIdentifier and a CwmsId?

**Review by rma-bryson — CHANGES_REQUESTED** [review]

File: cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/LocationIdentifier.java (hunk 1 above)
Comment (adamkorynta) [reply cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/LocationIdentifier.java:37]: You're right! removed

File: cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/TurbineEndpointInput.java [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        static final String OFFICE_QUERY_PARAMETER = "office";
+        private final String turbineId;
+        private final String officeId;
+
+        private GetOne(String turbineId, String officeId) {
+            this.turbineId = Objects.requireNonNull(turbineId, "Id required for getOne turbine endpoint");
+            this.officeId = Objects.requireNonNull(officeId, "Office id required for getOne turbine endpoint");
+        }
+
+        public String turbineId() {
```
[the remark this answers is not in the document]
Comment (adamkorynta) [reply cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/TurbineEndpointInput.java:92]: reduced scope

**Review by rma-bryson — DISMISSED** [review]

**Review by rma-bryson — APPROVED** [review]

**Pull request merged**